extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -143,15 +158,22 @@ export class AmpSidebar extends AMP.BaseElement {
    */
   close_() {
     this.viewport_.restoreOriginalTouchZoom();
-    this.mutateElement(() => {
+    this.vsync_.mutate(() => {
       this.closeMask_();
       this.element.removeAttribute('open');
       this.element.setAttribute('aria-hidden', 'true');
-      setStyles(this.element, {
-        'display': 'none',
-      });
       this.viewport_.removeFromFixedLayer(this.element);
     });
+    timer.delay(() => {
+      if (!this.isOpen_()) {
+        this.mutateElement(() => {
```

Comment:
No need for `mutateElement` - it should likewise be a vsync. You can only use `mutateElement` when you can capture both: before and after change states.